contribs/gmf/src/controllers/abstractdesktop.js

```diff
@@ -3,7 +3,7 @@ goog.provide('gmf.AbstractDesktopController');
 goog.require('gmf');
 goog.require('gmf.AbstractController');
 /** @suppress {extraRequire} */
-goog.require('gmf.mobileBackgroundLayerSelectorDirective');
+goog.require('gmf.mobilebackgroundlayerselectorDirective');
```

Comment:
I'm not sure but I think this is required so that a background layer is loaded.
See https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/apps/desktop/index.html#L15